Fixed #31750 -- Made models.Field equality compare models for inherited fields. (#13401)
django/django: docs/releases/3.2.txt

  instead of :exc:`~django.core.exceptions.SuspiciousOperation` when a session
  is destroyed in a concurrent request.

> * The :class:`django.db.models.Field` equality operator now compares also
>   models for inherited model field instances. Additionally, the ordering of
>   such fields is now defined.
> 

Reviewer: Maybe:

```suggestion
* The :class:`django.db.models.Field` equality operator now correctly 
  distinguishes inherited field instances across models. Additionally, the
  ordering of such fields is now defined.
```
